lib/auth/auth.go

```diff
@@ -1518,6 +1523,14 @@ func (a *Server) GenerateServerKeys(req GenerateServerKeysRequest) (*PackedKeys,
 	}
 
 	cert, signer, err := a.keyStore.GetTLSCertAndSigner(ca)
+	if trace.IsNotFound(err) {
```

Comment:
can we additionally limit to not issue server certs for external services?
for example, if an `ssh_service` joins the cluster while in this unrotated state, it should not get a cert signed by this auth server.

Reply:
We can do that. Would you think local services should be allowed (ie ssh_service within this auth server)? Can I just limit it to the `Admin` role only?

Reply:
I think limiting to the Admin and/or Auth roles should do it. Experiment, whatever lets `auth_service` start successfully and `tctl` to work.
AFAIK, when other services are colocated with Auth, it only issues one identity for itself and re-uses it for others.

Reply:
Okay, I've got it restricting to only the `Admin` role now. Local services are not able to connect, but that should actually be fine I guess, they just keep retrying to connect. `tctl` works.